udata_hydra/analysis/csv.py

```diff
@@ -97,6 +121,8 @@ async def analyse_csv(check_id: int = None, url: str = None, file_path: str = No
                 "parsing_finished_at": datetime.now(pytz.UTC),
             })
         await csv_to_db_index(table_name, csv_inspection, check)
+        if check_id:
```

Comment:
We could move these lines in the `finally` section, so that we also ping parsing error info